lib/components/ConnectModal/index.tsx

```diff
@@ -24,6 +24,11 @@ export const ConnectModal = (props: {
       document.body.style.overflow = "";
     }
   }, [showModal]);
+  useEffect(() => {
+    if (showModal) {
+      setStep("connect");
```

Comment:
just add this to L22 instead of making a second effect

Reply:
updated. sorry it took so long.